src/test/java/chalmers/pimp/model/ModelImplTest.java

```diff
@@ -122,4 +122,20 @@ void getAmountOfLayers() {
   void setSelectedTool() {
     assertDoesNotThrow(() -> model.setSelectedTool(null));
   }
+
+  @Test
+  void moveSelectedLayer() {
+    ILayer layer = LayerFactory.createRectangle(20, 15, 500, 70);
+    model.addLayer(layer);
+
+    model.selectLayer(0);
```

Comment:
Will be redundant call later after my commit. (if activeLayer is null and a new layer is created, that layer is now the active layer)